recipes/keyrings.alt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "keyrings.alt" %}
```

Comment:
Sure, but the pypi name does not conform to the conda (not conda-forge) spec. So that name cannot be guaranteed to work with conda.  Maybe they can change the name on pypi?